docs/quickstart/README.md

```diff
@@ -0,0 +1,198 @@
+# SRF Quick Start Guide (QSG)
+
+The SRF Quick Start Guide (QSG) provides examples on how to start using SRF via the Python bindings, C++ bindings or both.
+
+## Prerequisites
+
+- Pascal architecture or better
+- NVIDIA driver `450.80.02` or higher
+- [conda or miniconda](https://conda.io/projects/conda/en/latest/user-guide/install/linux.html)
+- if using docker:
+  - [Docker](https://docs.docker.com/get-docker/)
+  - [The NVIDIA container toolkit](https://docs.nvidia.com/datacenter/cloud-native/container-toolkit/install-guide.html#docker)
+
+## Getting Started
+
+To get started with the QSG, it is necessary to get the required SRF components before running any of the examples. The QSG supports two methods for getting the SRF components:
```

Comment:
Suggest listing these out before the list.

> ...two methods for getting the SRF components, Conda and building from source.

Reply:
agreed